doc-files/release-notes-20.0.1.md

```diff
@@ -0,0 +1,26 @@
+# Release Notes for JavaFX 20.0.1
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+These notes document the JavaFX 20.0.1 update release. As such, they complement the [JavaFX 20](https://github.com/openjdk/jfx20u/blob/master/doc-files/release-notes-20.md) release notes.
+
+## List of Fixed Bugs
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8299977](https://bugs.openjdk.java.net/browse/JDK-8299977)|Update WebKit to 615.1|web
+[JDK-8300954](https://bugs.openjdk.java.net/browse/JDK-8300954)|HTML default Range input control not rendered|web
+[JDK-8301022](https://bugs.openjdk.java.net/browse/JDK-8301022)|Video distortion is observed while playing youtube video|web
+[JDK-8301712](https://bugs.openjdk.java.net/browse/JDK-8301712)|[linux] Crash on exit from WebKit 615.1|web
+[JDK-8302294](https://bugs.openjdk.java.net/browse/JDK-8302294)|Cherry-pick WebKit 615.1 stabilization fixes|web
+[JDK-8302684](https://bugs.openjdk.java.net/browse/JDK-8302684)|Cherry-pick WebKit 615.1 stabilization fixes (2)|web
+[JDK-8303217](https://bugs.openjdk.java.net/browse/JDK-8303217)|Webview loaded webpage is not showing play, volume related buttons for embeded Audio/Video elements|web
+
+## List of Security fixes
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+JDK-8299781 (not public)|Improve JFX navigation|web
+JDK-8304177 (not public)|Unable to navigate to relative URLs after fix for JDK-8299781|web
```

Comment:
Oops. In my haste, I gave you the wrong Bug ID for this last one. It should be:

```
JDK-8303501 (not public)|Unable to navigate to relative URLs after fix for JDK-8299781|web
```

Reply:
Btw, in case you are interested, you can most easily see the two security bugs I pushed yesterday in the review branch I sent out:

https://github.com/kevinrushforth/jfx/compare/f28896aa63...cpu-2304-sync